builds/jenkins-build-pack-api/Jenkinsfile

```diff
@@ -899,7 +899,7 @@ def validateDeploymentConfigurations(def prop) {
       error "Wrong configuration. Value for Key 'providerRuntime' is missing in the configuration"
 
     } else {
-      def validRuntimes = ["nodejs4.3", "nodejs6.10", "nodejs8.10", "python2.7", "python3.6", "java8", "go1.x"]
+      def validRuntimes = ["nodejs8.10", "nodejs10.x", "python3.6", "java8", "go1.x"]
```

Comment:
Will this break build for services older runtimes?

Reply:
Addressed below.